### DISCUSSION TITLE
Deepgram SDK available for C++.

### DISCUSSION THREAD

**Discussion by rkoverov** [deepgram/community#730] [General help] [answered]
Is there a Deepgram SDK available for C++? Can you provide a sample code for streaming using Deepgram API with C++?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by nikolawhallon** [comment, marked as answer]
There is no SDK for C++, I did get DG streaming working back in the day in C++ with this library https://github.com/zaphoyd/websocketpp as part of a plugin to OBS: https://github.com/nikolawhallon/obs-deepgram-poc some of this code might help: https://github.com/nikolawhallon/obs-deepgram-poc/blob/main/WebsocketEndpoint.hpp

But  outside of a proof-of-concept, I never wrote anything up or made a minimal example. I might find time to do so, but in the meantime, maybe this helps point in the right direction? This is a pretty cool list of C++ WS libraries: https://github.com/facundofarias/awesome-websockets?tab=readme-ov-file#c-1
